content/applications/marketing/surveys.rst

```diff
@@ -1,6 +1,5 @@
 :nosearch:
 :show-content:
-:hide-page-toc:
 :show-toc:
```

Comment:
Would delete these too.
Structure it just like events so the top level menu link is a non-indexed page.